sigma/backends/loki/loki.py

```diff
@@ -881,6 +908,23 @@ def convert_condition_and(
             )
         )
 
+    def convert_condition_field_eq_field(
+        self, cond: SigmaFieldReference, state: ConversionState
+    ):
+        """
+        Constructs a condition that compares two fields in a log line to enable us to
+        search for logs where the values of two labels are the same.
+        """
+

```

Comment:
You're not using the field escaping here (see [this example](https://github.com/SigmaHQ/pySigma/blob/11ee93375b64b8c9f7f4a8e99042b50826057354/sigma/conversion/base.py#L1497) from pySigma) - is there a particular reason?

Reply:
Yep, good point, no reason for not using it. I didn't know that method was there to access that, I should have looked a little closer at the base